grpc-haskell.cabal

```diff
@@ -19,7 +19,7 @@ Flag Debug
 flag with-examples
   description: Also build example executables.
   manual:      True
-  default:     False
+  default:     True
```

Comment:
I changed it so I didn't have to look up the arcane syntax to override the flag at the command line. Then I forgot to change it back. I'll fix it.